cmd/tracee-ebpf/internal/config/config.go

```diff
@@ -0,0 +1,80 @@
+package config
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+)
+
+type TraceeEbpfCliConfig struct {
+	PerfRingBuffers struct {
+		EventSubmitBufferSize int
+		BlobBufferSize        int
+	}
+	VerboseDebug bool
+	InstallPath  string
+	BuildPolicy  string
+	Trace        []string
+	Output       []string
+	Capture      []string
+}
+
+var Config TraceeEbpfCliConfig
+
+func LoadConfig(fileName string) error {
```

Comment:
Why not return a `TraceeEbpfCliConfig` instance instead of defining globally?

Reply:
I wanted one instance of the config so it can be watched for value changes later, but maybe that was jumping ahead of it and could be implemented better anyways.
There was the suggestion to use viper package for that, but for now i'll change it to a return value.

Reply:
Done